test/controllers/pages_controller_test.rb

```diff
@@ -116,4 +116,26 @@ class PagesControllerTest < ActionDispatch::IntegrationTest
     @page.reload
     refute @page.file.attached?
   end
+
+  test "should create a zip file" do
+    req_url = "#{@page.url}.zip"
+    expected_zip_file = "tmp/#{req_url}"
+
+    get "/#{req_url}"
+
+    assert File.file?(expected_zip_file)
+    File.delete expected_zip_file
+  end
+
+  test "should download a zip file" do
+    req_url = "#{@page.url}.zip"
+    expected_zip_file = "tmp/#{req_url}"
+
+    get "/#{req_url}"
+
+    assert_equal @response.header["Content-Type"], "application/zip"
```

Comment:
I think we could add a file to the page and assert the content of the `.zip`, which must contain a file (png, pdf, etc) and a `.txt` file too. Can you add this?

Reply:
@MatheusRich Sure thing! Can I just add 1 more test for that, instead of adding it on the mentioned test? Will be better to see that way, IMHO.

Reply:
@wilbertliu No prob.